app/assets/javascripts/notes_feed/main.jsx

```diff
@@ -0,0 +1,27 @@
+import MixpanelUtils from '../helpers/mixpanel_utils.jsx';
+import NotesFeedPage from './NotesFeedPage.js';
+
+export default function renderNotesFeedMain(el, options = {}) {
+  if (options.json) {
```

Comment:
@gabivoicu Can you say a little more about the thinking behind using `options.json` here?

Reply:
@alexsoble Certainly: I copy pasted from somewhere else and not really understand how it works. Was just trying to figure it out now so I can write some tests. Do you have any good resources that explain how that works?

Reply:
@gabivoicu This was introduced in the school overview page when we were testing out a couple of ways of rendering data to the page. One way was to render data in an HTML data tag and the other was to use a JSON endpoint. We created an "options" path so that we could test out both approaches. We eventually went with the JSON endpoint but the old "options" code is still there. We should probably take it out now since it's confusing without context!